kedro_kubeflow/cli.py

```diff
@@ -276,8 +276,8 @@ def init(ctx, kfp_url: str, with_github_actions: bool):
     context_helper = ctx.obj["context_helper"]
     project_name = context_helper.context.project_path.name
     if with_github_actions:
-        image = f"gcr.io/${{google_project_id}}/{project_name}:${{commit_id}}"
-        run_name = f"{project_name}:${{commit_id}}"
+        image = f"gcr.io/${{oc.env:KEDRO_CONFIG_GOOGLE_PROJECT_ID}}/{project_name}:${{oc.env:KEDRO_CONFIG_COMMIT_ID, unknown-commit}}"  # noqa: E501
```

Comment:
@Lasica Shouldn't this be just `oc.env:google_project_id` as we are replacing the env variable's prefix and converting to lowercase [here](https://github.com/getindata/kedro-kubeflow/blob/f222f81182fb918311fe15cf0cf04a1120ff9bbf/kedro_kubeflow/context_helper.py#L55) ?

Further reference: [kedro-kubeflow doc](https://kedro-kubeflow.readthedocs.io/en/0.7.4/source/02_installation/02_configuration.html#dynamic-configuration-support)

Reply:
It used to be the case, now it's a bit irrelevant. It should work in the way I specified with the env variables provided. In general I want to remove and deprecate EnvTemplatedConfigLoader and replace it with recommendations to use oc.env directly.